alpaca/Dockerfile

```
@@ -1,21 +1,25 @@
 # syntax=docker/dockerfile:1.5.1
+FROM gradle:6.9.4-jdk11 AS builder
+
+ARG ALPACA_BRANCH="x-islandora-event-header"
```

Will need to set this back before merging

```suggestion
ARG ALPACA_BRANCH="2.x"
```